massgen/subagent/launch_watcher.py

```
@@ -0,0 +1,367 @@
+            task = asyncio.create_task(
+                self._handle_request_file(req_file, subagent_id),
+                name=f"SubagentLaunchWatcher.handle.{subagent_id}",
+            )
+            self._active_tasks[subagent_id] = task
+
+        # Clean up completed tasks
+        done_ids = [sid for sid, t in self._active_tasks.items() if t.done()]
+        for sid in done_ids:
+            del self._active_tasks[sid]
+
+    async def _handle_request_file(self, req_file: Path, subagent_id: str) -> None:
```

Ruff (0.15.2)

[warning] Unused method argument: `subagent_id`

(ARG002)